docs/send-data/hosted-collectors/cloud-to-cloud-integration-framework/onelogin-source.md

```diff
@@ -0,0 +1,101 @@
+
+## Data collected
+
+The data will be collected from OneLogin database using the following log:
+
+| Polling Interval | Data |
+| :--- | :--- |
+| 24 hours | [Users List](https://developers.onelogin.com/api-docs/2/users/list-users) |
+
+## Setup
+
+### Vendor configuration
+
+The OneLogin source requires you to provide the **Account URL**, **Client ID**, and **Client Secret** to set up the integration.
+
+#### Sub Domain
+
+The Sub Domain is the account URL used to retrieve the source data from the OneLogin API. For example, `https://{subdomain}.onelogin.com`.
+[Learn more](https://developers.onelogin.com/api-docs/2/getting-started/dev-overview) to obtain the sub domain URL.
+
+#### Client ID and Client Secret
+
+To generate the Client Id and Client Secret, see [Creating an API Credential Pair](https://developers.onelogin.com/api-docs/2/getting-started/working-with-api-credentials).
+
+Once you have all the required values, set up the source configuration to collect your desired log types available in the configuration section.
+
+### Source configuration
+
+When you create a OneLogin source, you add it to a Hosted Collector. Before creating the Source, identify the Hosted Collector you want to use or create a new Hosted Collector. For instructions, see [Configure a Hosted Collector and Source](/docs/send-data/hosted-collectors/configure-hosted-collector).
+
+To configure a OneLogin source:
+1. [**Classic UI**](/docs/get-started/sumo-logic-ui-classic). In the main Sumo Logic menu, select **Manage Data > Collection > Collection**. <br/>[**New UI**](/docs/get-started/sumo-logic-ui). In the Sumo Logic top menu select **Configuration**, and then under **Data Collection** select **Collection**. You can also click the **Go To...** menu at the top of the screen and select **Collection**.
+1. On the Collection page, click **Add Source** next to a Hosted Collector.
+1. Search for and select **OneLogin**.
+1. Enter a **Name** for the Source. The description is optional.
+1. (Optional) For **Source Category**, enter any string to tag the output collected from the source. Category metadata is stored in a searchable field called `_sourceCategory`.
+1. (Optional) **Fields**. Click the **+Add** button to define the fields you want to associate. Each field needs a name (key) and value.
+   * ![green check circle.png](/img/reuse/green-check-circle.png) A green circle with a check mark is shown when the field exists in the Fields table schema.
+   * ![orange exclamation point.png](/img/reuse/orange-exclamation-point.png) An orange triangle with an exclamation point is shown when the field doesn't exist in the Fields table schema. In this case, an option to automatically add the nonexistent fields to the Fields table schema is provided. If a field is sent to Sumo Logic that does not exist in the Fields schema it is ignored, known as dropped.
```

Comment:
this template was just updated last night https://github.com/SumoLogic/sumologic-documentation/pull/5634

```suggestion
   * ![green check circle.png](/img/reuse/green-check-circle.png) A green circle with a check mark is shown when the field exists and is enabled in the Fields table schema.
   * ![orange exclamation point.png](/img/reuse/orange-exclamation-point.png) An orange triangle with an exclamation point is shown when the field doesn't exist, or is disabled in the Fields table schema. In this case, you'll see an option to automatically add or enable the nonexistent fields to the Fields table schema. If a field is sent to Sumo Logic but isn’t present or enabled in the schema, it’s ignored and marked as **Dropped**.
```